src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.querySelector('.gallery__large-img');
+const listlImg = document.querySelector('ul');
+
+listlImg.addEventListener('click', (event) => {
+  event.preventDefault();
+
+  const image = event.target.closest('.list-item');
```

Comment:
here you can search for list-item__link (in this case you don't need to use children[0])